google-cloud-spanner/pom.xml

```diff
@@ -455,6 +455,24 @@
       <artifactId>opentelemetry-sdk-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.opentelemetry</groupId>
+      <artifactId>exporter-trace</artifactId>
+      <version>0.33.0</version>
```

Comment:
Please remove the version from here. It should either fetch from bom or the latest version